src/python/zquantum/core/circuits/_gates.py

```diff
@@ -1,28 +1,15 @@
 """Data structures for ZQuantum gates."""
 import math
 from dataclasses import dataclass, replace
-from functools import singledispatch
-from numbers import Number
 from typing import Callable, Dict, Iterable, Tuple, Union
 
 import numpy as np
 import sympy
 from typing_extensions import Protocol, runtime_checkable
 
+from ._operations import _get_free_symbols, Parameter, _sub_symbols
```

Comment:
`_get_free_symbols` and `_sub_symbols` are no longer fileprivate, should the leading underscore be removed?